generate_doc.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+handsdown
```

Comment:
```suggestion
handsdown mediahaven
```
Only create docs for the files in the `mediahaven` folder. Without it, it also tried to generate docs for certain packages in my `venv` folder.